sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -62,8 +62,9 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
       val testConfs = Seq(
         ("spark.sql.streaming.stateStore.providerClass",
           classOf[RocksDBStateStoreProvider].getName),
-        (RocksDBConf.ROCKSDB_CONF_NAME_PREFIX + ".compactOnCommit", "true"),
-        (RocksDBConf.ROCKSDB_CONF_NAME_PREFIX + ".lockAcquireTimeoutMs", "10")
+        (RocksDBConf.COMPACT_ON_COMMIT_CONF.fullName, "true"),
```

Comment:
Changed this test to show the bug of `ConfEntry.fullName`: SQLConf is always case sensitive but currently `ConfEntry.fullName` returns an incorrect lower case name in the error message.

This test will fail if we add `toLowerCase` back to `ConfEntry.fullName`.

Reply:
Should we remove this as well in RocksDBConf if we want to have consistent behavior, "case sensitive"?
`val confs = CaseInsensitiveMap[String](storeConf.confs)`